### ISSUE TITLE
a way of playing latest podcast episodes?

### ISSUE THREAD

**Issue by tomelabs** [jishi/node-sonos-http-api#648]
Hi there, 

I really love this project! thanks very much @jishi for your terrific work.
I got it running on a raspberry pi today and all seems to be going well.
This is possibly more of a sonos question, but I'm wondering if anyone has figured out how to trigger the latest episode of a given podcast to be played, without needing to nominate the individual podcast episode?

It seems that I can add a podcast at the title level to my Sonos Favourites (from several services, including Tune In or Pocketcasts), but these titles don't show up in my http://raspberrypi:5005/kitchen/favorites view 

Basically I'd like to be able to create a button that plays whatever the latest episode of a nominated podcast is. If anyone has got this workflow happening, please comment :-)
